src/test/java/hudson/markup/MyspacePolicyTest.java

```diff
@@ -40,10 +40,10 @@ public void testPolicy() {
         assertReject("script","<script src='http://foo/evil.js'></script>");
         assertReject("script","<script src='relative.js'></script>");
 
-        assertIntact("<style>H1 { display:none; }</style>");
+        assertReject("style", "<style>H1 { display:none; }</style>");
```

Comment:
```suggestion
        assertReject("style", "<style>H1 { display:none; }</style>");
        assertReject("style", "<div style='background-color: expression(alert(123));'>inline CSS</div>");
```

Tested locally, it passes :)